app/controllers/api/v1/crops_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::CropsController < ApplicationController
+
+  # GET /api/v1/crops
+  def index
```

Comment:
I suggest that for now we just got as minimal as we can.  Since we're only including ID/name, let's remove the sort params and all that other stuff that's included here.